spec/use_cases/source_tag_spec.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe EsaFeeder::UseCases::SourceTag do
+  let(:tags) { (0..6).map { |n| described_class.new( Time.local(2017, 1, 1 + n) ).call} }
+  let(:ref) {['sun', 'mon', 'tue', 'wed', 'thu', 'fri', 'sat']}
```

Comment:
__[RuboCop]__

Style/WordArray: Use `%w` or `%W` for an array of words.


<!-- Comment by [SideCI](https://sideci.com) -->